add backup-integration in Core

ArveVM opened the pull request:
Not sure if link's are OK,, did not get to view/check them in preview - am I doing something wrong?

## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Add info on system migration, 
- added restore for supervised-backup 
- Added info on backup-integration in core/container
- removed specifics on hidden files/folders



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

ArveVM committed:
adding backup-integration in Core

info on system migration, added restore for supervised-backup
Added info on backup-integration in core/container - removed specifics on hidden files/folders

Not sure if link's are OK,, did not get to view/check them in preview - am I doing something wrong?

frenck commented on source/_docs/configuration.markdown:
```diff
@@ -40,6 +40,7 @@ If you can't see your integration listed there, you will need to restart Home As
 ## Migrating to a new system
 
-The preferred way of migrating to a new system is by {% my supervisor_backups title="making a backup" %}.
+The preferred way of migrating to a new system is by backup using {% my supervisor_backups title="supervisor backup" %}. Then install new system (HAOS or Supervisor), and restore backup at first logon.
```
I don't think changing the link to "supervisor backup" is correct, it might be confused with taking a backup of just the supervisor (which is of course not the case).

```suggestion
The preferred way of migrating to a new system is by backup using {% my supervisor_backups title="backup" %}. Then install the new system (HAOS or Supervisor), and restore the backup during the onboarding steps.
```

frenck commented on source/_docs/configuration.markdown:
```diff
@@ -40,6 +40,7 @@ If you can't see your integration listed there, you will need to restart Home As
+The preferred way of migrating to a new system is by backup using {% my supervisor_backups title="supervisor backup" %}. Then install new system (HAOS or Supervisor), and restore backup at first logon.
 
-If you run the container or core installation methods, you will need to manually make a backup of your configuration folder. Be aware that some of the files you need start with `.`, which is hidden by default from both `ls` (in SSH), in Windows Explorer, and macOS Finder. You'll need to ensure that you're viewing all files before you copy them.
+If you run the container or core installation method you don't have supervisor-backup and will need to use the {% my backup title="backup-integration" %} to make a backup.
```
I now see the problem you are trying to solve with the "supervisor backup". Maybe instead, we should write something like:

> The preferred way of migrating to a new system is by making a backup. Backups can be found and created in the Settings menu, next go to **System** -> **Backup**.

The my links might be more confusing, so maybe just leaving them out, in this case, is an option?

frenck commented:
@ArveVM I'm confused why you requested a review. There are comments open. Additionally, when done, use the ready for review button. As the bot responded above, follow the link to read more about our review process.

Thanks 👍 

../Frenck

ArveVM committed:
Merge branch 'current' into patch-4

ArveVM committed:
removed links - simplified

added wording from your comment on backup location
simplified the specification of restore    - could add link to installation-method,, but KISS?

Andsorry for the "send to review",, just starting to contribute on documentation - and have not used github or other developers tools. 
I've been challenged by peers to contribute, and that "it is easy,, you will get plenty of good guidance" ;)  - so I just took a shot :)
Have now tried to read a bit on tpe PR-process, so hopefully my contributions from now on will follow more the standard process. - and sorry for the inconvenience/extra work I've created for you (on 2 out of 3 PR's so far :)  )
 - I stribe to do better  :)
